test/functional-tests/uploader.spec.js

```diff
@@ -32,6 +32,7 @@ describe('Uploader Tool', function () {
     assert.isOk(videoId)
     assert.isOk(videoId.length > 8)
     // set title and video in the form
+    browser.pause(1000)
```

Comment:
Let’s work on an alternative solution. Pause killed us in the last app 😅

Reply:
yes i know, was just a temp solution. i agree with you!